courses/machine_learning/deepdive2/structured/solutions/5a_train_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,1593 @@
+{
```

Comment:
<p>just a note....</p><p>if they do get errors below then this work around won't work since just cloning the repo then navigating to the solutions directory won't have the necessary files they would have just created in their clone. </p><p> </p><p>Two ways to fix this come to mind</p><ol><li>include all the ancillary Docker files and model.py and task.py in the solutions folder of the repo so the above push_docker.sh will work</li><li>we could have then fork the repo to start and they could push their changes to their fork , then cloning their fork in CloudShell above would work....with obvious modifications to the repo name. (more confusing?)</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/674/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>Ya, these are the instructions Lak created so I just copied it over. I can create those files and add them to the commit so that they can just use them in Cloud Shell.</p>